Automattic/newspack-plugin
# fix(ras-acc): remove redundant email template alt attributes

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://app.asana.com/0/1207817176293825/1207990881852216/f

This PR removes some residual alt attributes from images that were specific to my test site that were causing the following text to appear in email preview text in gmail: `acof-flag`

### How to test the changes in this Pull Request:

This one will require your test site to be able to send emails to your gmail account. If you can't do this, I've set up just the OTP email template with these changes on ras-acc.newspackstaging.com.

1. As admin, go to Newspack > Engagement, select Show Advanced Settings, then scroll down to the transactional emails section
2. Select the reset button next to the OTP email template
3. As a reader attempt to login via OTP
4. Check gmail and verify the email preview text doesn't contain the image alt text `acof-flag`

![Screenshot 2024-08-08 at 11 46 37](https://github.com/user-attachments/assets/44720e43-825f-4f7a-8e85-bbd771c4f991)

### Other information:
